code/script.js

```diff
@@ -1,325 +1,396 @@
+      other: [],
+    },
+    {
+      name: "Josephine",
+      img: "images/josephine.svg",
+      hair: "grey",
+      eyes: "brown",
+      accessories: [],
+      other: [],
+    },
+    {
+      name: "Josh",
+      img: "images/josh.svg",
+      hair: "yellow",
+      eyes: "green",
+      accessories: [],
+      other: [],
+    },
+    {
+      name: "Jude",
+      img: "images/jude.svg",
+      hair: "black",
+      eyes: "green",
+      accessories: [],
+      other: [],
+    },
+    {
+      name: "Julie",
+      img: "images/julie.svg",
+      hair: "black",
+      eyes: "brown",
+      accessories: ["glasses", "hat"],
+      other: [],
+    },
+  ];
+
+  // Global variables
+  let secret;
+  let currentQuestion;
+  let charactersInPlay;
+  
+
+  // Draw the game board
+  const generateBoard = () => {
+    board.innerHTML = "";
+    charactersInPlay.forEach((person) => {
+      board.innerHTML += `
+        <div class="card">
+          <p>${person.name}</p>
+          <img src=${person.img} alt=${person.name}>
+          <div class="guess">
+            <span>Guess on ${person.name}?</span>
+            <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+          </div>
         </div>
-      </div>
-    `
-  })
-}
+      `;  
+    });
+  };
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
-const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  // Randomly select a person from the characters array and set as the value of the variable called secret
+  const setSecret = () => {
+    secret =
+      charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+  };
 
-// This function to start (and restart) the game
-const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
-}
+  // This function to start (and restart) the game
+  const start = () => {
+    charactersInPlay = CHARACTERS; // Here we're setting charactersInPlay array to be all the characters to start with
+    document.getElementById('winOrLose').style.display = "none";
+    generateBoard(); //call/invoke function generateBoard on row 210
+    setSecret(); //call/invoke function setSecret on row 227
+    console.log("secret:", secret);
+  };
 
-// setting the currentQuestion object when you select something in the dropdown
-const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
+  // setting the currentQuestion object when you select something in the dropdown
+  const selectQuestion = () => {
+    const category =
+      questions.options[questions.selectedIndex].parentNode.label;
+    //This variable stores what option group (category) the question belongs to.
+    const value = questions.value;
```

Comment:
this was nice that you could make the code so short. I declared it like this: 
```const value = questions.options[questions.selectedIndex].value```
but your looks more neat.